### PR TITLE
feature description

### PR DESCRIPTION
**Pull request by toropeza**
This commit allows moderators to add a description for apps that are featured. Let me know if there are recommendations for the design.

A working version can be found at http://fdesc.aigallerythomas.appspot.com/
Changes made:
-New styling on Featured app list
screen shot 2016-02-09 at 3 37 58 pm

-Featured description shows on GalleryPage
screen shot 2016-01-26 at 4 57 09 pm
-Popup panel for adding the description
screen shot 2016-01-26 at 4 57 24 pm

Thank you,
Thomas Oropeza

### CODE REVIEW

**Comment by toropeza** [comment]
@wolberd @jisqyv

File: appinventor/appengine/war/Ya.css [hunk 1]
```diff
@@ -1676,6 +1676,45 @@ textarea::-webkit-input-placeholder {
 .gallery-card {
   width: 160px;
 }
+.gallery-featured-title {
+    font-size:25px;
```
Comment (josmas) [inline appinventor/appengine/war/Ya.css]: indentation needs fixing

File: appinventor/appengine/src/com/google/appinventor/client/explorer/youngandroid/GalleryPage.java [hunk 2]
```diff
@@ -103,6 +105,7 @@
   private FlowPanel appComments;
   private FlowPanel appCommentsList;
   private FlowPanel returnToGallery;
+  private PopupPanel featureAppPopupPanel;
```
Comment (josmas) [inline appinventor/appengine/src/com/google/appinventor/client/explorer/youngandroid/GalleryPage.java]: could you please delete commented out code? it's obviously not being used.

File: appinventor/appengine/src/com/google/appinventor/client/GalleryGuiFactory.java [hunk 3]
```diff
@@ -181,6 +181,103 @@ public void onClick(ClickEvent event) {
...
+   * @param refreshable: if true then the GUI can be reloaded later.
+   */
+  public void generateFeaturedAppList(List<GalleryApp> apps,
+                                        FlowPanel container, Boolean refreshable) {
+    if (refreshable) {
+      // Flush the panel's content if we knew new stuff is coming in!
+      container.clear();
+    }
+    for (final GalleryApp app : apps) {
+      // Create the associated GUI object for app
```
Comment (josmas) [inline appinventor/appengine/src/com/google/appinventor/client/GalleryGuiFactory.java]: There are a number of text comments that could be deleted, such as this line, it doesn't really add much.

There's also a number of code lines commented out that should also go. Two in particular are override annotations; is the code overriding or not?

**Comment by toropeza** [comment]
@josmas @wolberd @jisqyv 
Changes made in accordance with review.

A couple of additions were made also

1. The featured description will now be cut off in the Gallery Featured List view if it is too long.
2. Added a Gold Border to the Featured Apps making them stand out a little more (Let me know what you think about that)

**Comment by jisqyv** [comment]
@toropeza Is there a reason why you closed this?

**Comment by toropeza** [comment]
Yes I want to change some parts and update it with newest master.

Thank You,
Thomas Oropeza

**Pull request closed without merging**